patch_rebaser/patch_rebaser.py

```diff
@@ -228,10 +228,18 @@ def perform_rebase(self):
         try:
             LOGGER.info("Rebasing %s to %s", self.branch, self.commit)
             self.repo.branch.rebase_to_hash(self.branch, self.commit)
-        except git_exceptions.RebaseException:
-            LOGGER.info("Could not rebase. Cleaning up.")
-            self.repo.branch.abort_rebase()
-            raise
+        except git_exceptions.RebaseException as e:
+            if '.gitreview' in str(e):
+                LOGGER.warning("A patch including the .gitreview file failed "
+                               "to rebase, skipping it.")
+                self.repo.git.rebase('--skip')
+                self.perform_rebase()   # Just in case, rebase again
```

Comment:
Do we want to wrap this in a try...except as well just to catch any other rebase issues? We may need to consider having a loop of; rebase, auto-mitigate, continue until we either hit an issue where we can't auto-mitigate or we complete the rebase. That would require more changes, and would be out of the scope of this PR, but something we could do as a next step.

Reply:
Agree, I think as a next step this process needs to be decomposed.  Actually, I was just thinking, maybe we split out the error handling to a separate method as part of that.  This code feels to me like the beginnings of a decision tree, and as soon as we consider adding one more if here, we should reassess and make such a mechanism, imo.